apps/payments/next/providers/index.tsx

```diff
@@ -0,0 +1,33 @@
+'use client';
```

Comment:
```suggestion
/* This Source Code Form is subject to the terms of the Mozilla Public
 * License, v. 2.0. If a copy of the MPL was not distributed with this
 * file, You can obtain one at http://mozilla.org/MPL/2.0/. */

'use client';
```

Reply:
Ah thank you! I keep forgetting for new files! :homerdisappear: :mozheart:

Fixed!